mod-linked-data/src/main/resources/citation/mod-linked-data/restart-mod-search-kafka-listeners.feature

```diff
@@ -0,0 +1,13 @@
+Feature: mod-linked-data integration tests
+
+  Background:
+    * url baseUrl
+    * table searchModule
+      | name         |
+      | 'mod-search' |
+
+  Scenario: Restart mod-search kafka listeners
+    // Create a dummy tenant & enable mod-search for that tenant. Also, delete that tenant immediately.
```

Comment:
mod-search will restart the kafka listeners for all tenants when a new tenant is initialized. Hence, this approach will work.